src/middlewares/validate-new-user.js

```diff
@@ -0,0 +1,31 @@
+import db from '../models';
+
+export default async (req, res, next) => {
+  const errors = {};
+  if (Object.keys(req.body).length <= 0) {
+    return res.status(400).json({
+      errors: {
+        body: ['body can not be empty']
+      }
+    });
+  }
+
+  const checkUser = await db.User.findAll({
+    where: {
+      email: req.body.email
+    },
+    logging: false
+  });
+
+  if (checkUser.length > 0) {
```

Comment:
This can be moved into a validator

Reply:
Do you mean that this should also be defined in a helper and imported?

Reply:
yeah, we can have a `doesUserExist` helper so that if ever anyone else needs it they can call that helper

Reply:
ok